<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraDescription</class>
 <widget class="QDialog" name="CameraDescription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Camera Description</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt_pco_camera.qrc">
    <normaloff>:/Resources/PCOApp.ico</normaloff>:/Resources/PCOApp.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>80</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="6" column="1">
      <widget class="QLabel" name="SensorType">
       <property name="text">
        <string>Sensor</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="SensorTypeLabel">
       <property name="text">
        <string>Sensor Type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="SerialNumber">
       <property name="text">
        <string>Number</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="CameraTypeLabel">
       <property name="text">
        <string>Camera Type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="Temperature">
       <property name="text">
        <string>0Â°</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="TemperatureLabel">
       <property name="text">
        <string>Temperature:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="SerialNumberLabel">
       <property name="text">
        <string>Serial Number:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="CameraType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="PixelRate">
       <property name="text">
        <string>0</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0" alignment="Qt::AlignTop">
      <widget class="QLabel" name="PixelRateLabel">
       <property name="text">
        <string>Pixelrates:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="Resolution">
       <property name="text">
        <string>0 x 0</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="ResolutionLabel">
       <property name="text">
        <string>Resolution:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="CameraHealthLabel">
       <property name="text">
        <string>Camera Health:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="CameraHealth">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
      <widget class="QLabel" name="FirmwareLabel">
       <property name="text">
        <string>Firmware:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="Firmware">
       <property name="text">
        <string>Version</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="CameraDescriptionOkButton">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>41</height>
        </size>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_pco_camera.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CameraDescriptionOkButton</sender>
   <signal>clicked()</signal>
   <receiver>CameraDescription</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
